tracing-futures/src/lib.rs

```diff
@@ -87,41 +87,44 @@ pub trait WithSubscriber: Sized {
 }
 
 /// A future, stream, or sink that has been instrumented with a `tracing` span.
+#[cfg_attr(feature = "std-future", pin_project)]
 #[derive(Debug, Clone)]
 pub struct Instrumented<T> {
+    #[cfg(feature = "std-future")]
+    #[pin]
+    inner: T,
+    #[cfg(not(feature = "std-future"))]
     inner: T,
     span: Span,
 }
 
 /// A future, stream, sink, or executor that has been instrumented with a
 /// `tracing` subscriber.
+#[cfg_attr(feature = "std-future", pin_project)]
 #[derive(Clone, Debug)]
 pub struct WithDispatch<T> {
+    // cfg_attr doesn't work inside structs, apparently...
```

Comment:
Oh, this is related to the order of application of proc-macro (https://github.com/taiki-e/pin-project/issues/68#issuecomment-528318870). I will fix this.